common/dynamicconfig/file_based_client_test.go

```diff
@@ -183,17 +179,16 @@ func (s *fileBasedClientSuite) TestGetMapValue() {
 }
 
 func (s *fileBasedClientSuite) TestGetMapValue_WrongType() {
-	var defaultVal map[string]interface{}
```

Comment:
may be a sign we shouldn't use default values (for trivial ones, bool is fine) in these tests, to ensure we're not hitting some other kind of fallback branch